features/facebookImport/src/model/entities/utils/entity-serialisation.js

```diff
@@ -0,0 +1,43 @@
+import { dataImporters } from "../../importer";
+import ImporterExecutionResult from "../../importers/utils/importer-execution-result";
+import FacebookAccount from "../facebook-account";
+import MessageThread from "../message-thread";
+import MessageThreadsGroup from "../message-threads-group";
+import RelatedAccount from "../related-account";
+import RelatedAccountsGroup from "../related-accounts-group";
+import RelatedPost from "../related-post";
+
+const KnownClasses = [
```

Comment:
Why do we need to have this? Wouldn't it be better to just have some role or method?